packages/documentation/src/stories/utilities/heading.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
+
+<Meta
+  title="Utilities/Heading"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+  }}
+/>
+
+# Headings
+
+<p className="lead">
+  Use the heading utilities, to match the font styling of a heading without the need of the
+  associated HTML element.
+</p>
+
+<div className="alert alert-warning">
+  <p>
+    We point out that semantic headings are important for{' '}
+    <a href="/?path=/docs/foundations-accessibility--page">accessibility</a>.
+  </p>
+  <p>Use these classes only in exceptional cases.</p>
```

Comment:
```suggestion
  <p>
    ```suggestion
    Never skip a heading level. Maintaining a semantically correct heading structure helps users of assistive technologies and search engines understand your content. Sometimes the correct heading level has the wrong styles for the design. In this case, you can make use of the heading level helper classes. Structure your headings semantically correct and just change the look of the text.
  </p>
  <p>
    Find more information on our{' '}
    <a href="/?path=/docs/foundations-accessibility--page">accessibility page</a>.
  </p>
```